shell/platform/windows/fixtures/main.dart

```diff
@@ -159,6 +159,37 @@ void enableLifecycleToFrom() async {
   });
 }
 
+@pragma('vm:entry-point')
+void sendCreatePlatformViewMethod() async {
+  // The platform view method channel uses the standard method codec.
+  // See https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/services/message_codecs.dart#L262
```

Comment:
I would use a permalink here. The line number will be wrong as the code continues to evolve